index.html

```diff
@@ -46,6 +46,207 @@ <h1>I'm Naan.
+                                <div class="see_project">See project</div>
+                            </button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div>
+                <div class="about_myself">
+                    <div class="top underline">
+                        <div class="about_header underline">About Myself</div>
+                        <div class="about_description ">
+                            Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
+                            If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                        </div>
+                        <div class="social_media">
+                            <div class="supporting_text underline">
+                                <div class="connect ">LET’S CONNECT</div>
+                            </div>
+                        </div>
+                        <div class="icon_container border">
+                            <div><img src="images/github.jpg"></div>
+                            <div><img src="images/linkedin.jpg"></div>
+                            <div><img src="images/reddit.jpg"> </div>
+                            <div><img src="images/twitter.jpg"> </div>
+                            <div><img src="images/medium.jpg"> </div>
+                        </div>
+                        <div class="resume_button">
+                            <div class="resume_text underline">Get my resume</div>
+                        </div>
+                    </div>
+                    <div class="skills_list">
+                        <div class="languages">
+                            <div class="language_header underline ">
+                                <p>Languages</p>
+                                <img src="images/lang_arrow.png">
+                            </div>
+                            <div class="language_list border">
+                                <div class="lang "><img class="round_image" src="images/javascript.jpg">&nbsp; Javascript</div>
+                                <div class="lang "><img class="round_image" src="images/html.jpg">&nbsp; HTML</div>
+                                <div class="lang "><img class="round_image" src="images/css.jpg">&nbsp; CSS</div>
+                            </div>
+                        </div>
+                        <div class="text_arrow underline">
+                            <p>Frameworks</p>
+                            <img src="images/Union.png">
+                        </div>
+                        <div class="text_arrow underline">
+                            <p>Skills</p>
+                            <img class="left" src="images/Union.png">
+                            <hr class="separator">
+                        </div>
+                    </div>
```

Comment:
- [ Optional ] It would be better if this part of your webpage uses `ul` and `li` tags since it's a listing of items on a page. The `ul` tag would make it easy for screen readers to identify a list on a webpage for users using web accessibility tools.

- [ Optional ] It would be better if this part of your design matches the `Figma` template provided, currently, the `Frameworks` and `Skills` sections are missing the `border-bottom` `CSS` rule that is present on the `Figma` template. Please see the screenshot below for more details on this.

## Figma template
![Screenshot from 2023-02-19 01-53-12](https://user-images.githubusercontent.com/16392046/219902989-0f5bf613-6710-41db-8b3a-2ef532c9a1f8.png)


## Your design
![Screenshot from 2023-02-19 01-41-56](https://user-images.githubusercontent.com/16392046/219902906-548f9f59-1c43-4128-aae5-06bf55b0040f.png)